Assets/Scripts/Player/PlayerHandDetection.cs

```diff
@@ -57,15 +57,16 @@ public class PlayerHandDetection : MonoBehaviour {
 		foreach (Interactable interactable in interactableList) {
 			float distance = (transform.position - interactable.transform.position).sqrMagnitude;
 
-			if (distance < lowestDistance) {
+			if (distance < lowestDistance || interactable.CompareTag(priorityInteractableTag)) {
 				lowestDistance = distance;
-				closestItem = interactable;
+				closestItem = interactable;				
```

Comment:
```suggestion
				closestItem = interactable;
```

Reply:
I think you are pointing to the tabs after the 62 line? X)